TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -2273,7 +2273,8 @@ as discussed in <doc:Declarations#Extension-Declaration>.
 ## Protocol Declaration
 
 A *protocol declaration* introduces a named protocol type into your program.
-Protocol declarations are declared at global scope
+Protocol declarations are declared either at global scope
```

Comment:
```suggestion
Protocol declarations are declared either at global scope,
```

Reply:
Rewrote this — there are too many clauses here, and joining them with commas is getting awkward.